index.html

```diff
@@ -0,0 +1,37 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <link rel="stylesheet" href="css/style.css">
+  </head>
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <body>
+    <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
+
+    <body class="bk-Color">
+      <div id="navbar">
+        <a class="active" href="#home">Home</a>
+        <a href="#projects">Projects</a>
+        <a id="profile-link" href="https://github.com/mateo951/TrialsJM" target="_blank">GitHub repo</a>
+      </div>
```

Comment:
- [ ] Due to semantic reasons, it's considered a good practice to use a `<header>`  because you should always aim to describe a peace of content the best way you can with a proper tag. A `<div>` does portray a division of content but lacks specification on which type of content you will find in it. In this case,  `<header>`  [represents a container for introductory content or a set of navigational links](https://www.w3schools.com/tags/tag_header.asp) so its a much better fit!